### PR TITLE
Fix link and mention Zach's other projects.

### PR DESCRIPTION
**Pull request by sduckett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ A collection of tutorials on how to get started with Clojure using different edi
 
 * [Getting Started using Nightcode](nightcode.md)
 
-######Note that the getting started guide being used by ClojureBridge workshops is [here](https://github.com/ClojureBridge/curriculum/blob/master/outline/setup.md).
+######Note that the getting started guide being used by ClojureBridge workshops is [here](https://github.com/ClojureBridge/curriculum/blob/gh-pages/outline/setup.md).
```
Comment (iantruslove) [inline README.md]: Is the URL correct? Shouldn't this be the denver-clojurebridge org's fork of curriculum?

Comment (sduckett) [inline README.md]: Quite right

**Comment by iantruslove** [comment]
inc

**Comment by sduckett** [comment]
This should do it.

**Comment by iantruslove** [comment]
@sduckett I say merge it

**Pull request merged**